Sensor response low photoelectron signal function

bpalmeiro opened the pull request:
This function is meant to model the sensor responses when they work in avalanche mode.

jwaiton commented on packs/tools/fit_functions.py:
```diff
@@ -0,0 +1,24 @@
+import scipy.stats as scs
+
+
+def finger_signal(xs         : np.array,
```
Docstrings describing the functions could be useful. Perhaps it would be a good assignment to ask the students to do this?

jwaiton commented on packs/tools/fit_functions_test.py:
```diff
@@ -0,0 +1,19 @@
+       floats(min_value = -500, max_value = 500),
+       floats(min_value =    1, max_value =   2),
+       floats(min_value =    0, max_value = 200))
+def test_finger_signal_gaus_when_poisson_zero(bl, gain, sbl, sq):
```
likewise, documentation would be nice

jwaiton commented:
I've added docstrings, comments and  some alterations in logic.

I've also added this catch statement to avoid an issue when the poisson mu is set to zero.
https://github.com/nu-ZOO/MULE/blob/255261cc85e5038acf5a3547456b668a7f61ae05/packs/tools/fit_functions.py#L42

This doesnt feel elegant, and also possibly doesn't match what you'd expect.

bpalmeiro commented:
> I've added docstrings, comments and some alterations in logic.
> 
> I've also added this catch statement to avoid an issue when the poisson mu is set to zero.
> 
> https://github.com/nu-ZOO/MULE/blob/255261cc85e5038acf5a3547456b668a7f61ae05/packs/tools/fit_functions.py#L42
> 
> This doesnt feel elegant, and also possibly doesn't match what you'd expect.

My only concern with this (that is not decoupled from the previous implementation, I believe) is that when you do this, then sigmaq becomes a floating parameter, and that can be problematic when fitting, but I don't have a good solution rn sadly
